src/pages/DistributionsPage/DistributionsPage.tsx

```diff
@@ -172,15 +172,47 @@ const Summary = ({
       <Text css={{ color: '$complete' }}>
         Distribution submitted {formatRelative(distTime, Date.now())}
       </Text>
-      <Box css={{ display: 'flex', alignItems: 'center' }}>
-        <Icon css={{}} />
-        <Link
-          css={{ ml: '$xs' }}
-          href={`https://etherscan.io/tx/${distribution.tx_hash}`}
-        >
-          View on Etherscan
-        </Link>
-      </Box>
+      <ExplorerLink distribution={distribution} />
     </Box>
   );
 };
+
+const ExplorerLink = ({
+  distribution,
+}: {
+  distribution: EpochDataResult['distributions'][0];
+}) => {
+  const { tx_hash } = distribution;
+  const { chain_id } = distribution.vault;
+
+  const explorerHref = makeExplorerUrl(chain_id, tx_hash);
+
+  if (!explorerHref) return <></>;
+
+  return (
+    <Box css={{ display: 'flex', alignItems: 'center' }}>
+      <Icon css={{}} />
+      <Link css={{ ml: '$xs' }} href={explorerHref}>
+        View on Etherscan
+      </Link>
+    </Box>
+  );
+};
+
+function makeExplorerUrl(chainId: number, txHash: string | undefined) {
```

Comment:
I like this function. Only change would be to move it to the utils folder so that it can be reused. 1.5 designs have other Etherscan links but I don't think it's a blocker to this PR.